src/server/server.rs

```diff
@@ -59,6 +63,7 @@ impl<T: RaftStoreRouter, S: StoreAddrResolver + 'static> Server<T, S> {
         snapshot_status_sender: Sender<SnapshotStatusMsg>,
         resolver: S,
         snap_mgr: SnapManager,
+        engine: Option<Engines>,
```

Comment:
rename it to `engines_for_debug` or something? It's just used for debug_service.